[supervisord]
nodaemon=true
user=root
logfile=/dev/null
logfile_maxbytes=0

[eventlistener:update]
events=TICK_3600
directory=/app
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0
command=python update_app.py

[program:initial_update]
directory=/app
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0
autorestart=unexpected
exitcodes=0
command=python manage.py update_warehouse

[program:warehouse]
directory=/app
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0
command=/app/start_with_update.sh

[program:nginx]
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0
command=nginx -g 'daemon off; error_log /dev/stderr;'